string target_frame
---
int32[] ids
geometry_msgs/PoseStamped[] apriltag_poses
shape_msgs/SolidPrimitive[] shapes